### PR TITLE
Add interactive list

### PR DESCRIPTION
**Pull request by ivangonzalez224**
It's my To do list project and I added the following features:

- Added a module to clear task list function.
- Added event listener to check boxes to update tasks status.
- Added event listener to the clear all completed button to update task list.

### CODE REVIEW

File: src/index.html
Comment (Efiamotu-1) [inline src/index.html:1]: - [ ] Your usage section of your readme is not correctly written to enable user run the application locally please kindly see the example below and make the necessary changes. 👍 ✨ 
-  example
- > After installing the project, run the following command:
- run `npm run start` or `npm start` to open the project in your browser 
- * This is all that should be required of you to assist anyone using your app for the first time whether it is a developer or just a random user 👍

Comment (Efiamotu-1) [inline src/index.html:1]: - [ ] Your live demo section currently doesn't have the app demo, you can try deploying your project with `github pages` or if the link is still not available you can remove the section and only put that section if the link is available 👍

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @ivangonzalez224 , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- 
- ✔️ Good job implementing the interactive list
- ✔️ Good job using JavaScript `ES 6 Modules`
- ✔️ Great UI design
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ivangonzalez224** [comment]
```sh
  npm run start
```

File: src/index.html
Comment (peterrobert) [reply src/index.html:1]: This change has been done well

Comment (peterrobert) [reply src/index.html:1]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @ivangonzalez224, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**